src/backend/models/firefighter.js

```diff
@@ -1,9 +1,11 @@
 const mongoose = require("mongoose");
+const bcrypt = require("bcrypt");
 
 const FirefighterSchema = new mongoose.Schema(
   {
     name: String,
-    ff_id: String,
+    email: { type: String, unique: true, index: true },
```

Comment:
As discussed, we could keep the id here, which is the internal number of each firefighter, and only allow the one's I've added to the schema. We don't need emails for anything else, so we can probably ditch it

It can be done in another PR though

Reply:
I will refactor in the end, to remove email and use id